### PR TITLE
Event detail changes

### PR DESCRIPTION
**Pull request by odinuge**
Depends on https://github.com/webkom/lego/pull/1512

- Add a few more payment statuses
- Disable payment button during payment
- Show stripe payment button after event has started
- Update payment status real time
- Simplify registration form
- Move registration form to right side of screen

Screenshot from 2019-03-19 04-19-56
Screenshot from 2019-03-19 04-20-00
Screenshot from 2019-03-19 04-20-20
Screenshot from 2019-03-19 04-24-24

Screenshot from 2019-03-19 18-01-38
Screenshot from 2019-03-19 18-04-48
Screenshot from 2019-03-19 18-05-03
Screenshot from 2019-03-19 18-08-36
Screenshot from 2019-03-19 19-20-59
Screenshot from 2019-03-19 19-26-07

### CODE REVIEW

**Comment by odinuge** [comment]
Discussed a bit with @SmithPeder and we more or less agreed about moving the payment button (and maybe also the register button) to the right side over the admin section.

Out of scope of this tho

File: app/routes/events/components/StripeElement.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ class PaymentRequestForm extends React.Component<FormProps, State> {
 
         {this.state.canMakePayment === false && (
           <StripeCheckout
-            name="Abakus Linjeforening"
+            name="Abakus"
```
Comment (odinuge) [inline app/routes/events/components/StripeElement.js]: :trollface: @edvardvb

File: app/routes/events/components/RegistrationMeta.js [hunk 2]
```diff
@@ -38,16 +167,18 @@ const RegistrationMeta = ({
...
-            <div>
-              <i className="fa fa-check-circle" /> Du har betalt
-            </div>
-          ) : (
-            <div>
-              <i className="fa fa-exclamation-circle" /> Du har ikke betalt
-            </div>
-          ))}
+        <PresenceStatus presence={registration.presence} hasEnded={hasEnded} />
+        <Tooltip content="Du kan når som helst endre samtykket ved å kontakte oss på abakus@abakus.no">
```
Comment (odinuge) [inline app/routes/events/components/RegistrationMeta.js]: Hva tenker HS om noe ala. dette @JonasBak?

Comment (JonasBak) [reply app/routes/events/components/RegistrationMeta.js]: Jeg er fan, skal ta det opp med resten :bento:

**Comment by kotlarz** [comment]
💯 💯 💯 💯 
Awesome work @odinuge!

**Pull request merged**